xcode/Subconscious/Shared/UIKit/BlockEditor/BlockEditorRepresentable.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  BlockEditorRepresentable.swift
+//  SubconsciousExperiments
+//
+//  Created by Gordon Brander on 6/6/23.
+//
+
+import SwiftUI
+import Combine
+import os
+
+extension BlockEditor {
+    //  MARK: View Representable
```

Comment:
Linter is upset because of double space between `//` and `MARK`, not important but I think we use one space elsewhere?

Reply:
Fixed all instances of `//   MARK:`